src/test/mock/mock.sol

```diff
@@ -47,6 +60,14 @@ contract Mock {
         values_bool_return[name] = returnValue;
     }
 
+    function setBytes32Return(bytes32 name, bytes32 returnValue) public {
+        values_bytes32_return[name] = returnValue;
+    }
+
+    function setIntReturn(bytes32 name, int returnValue) public {
+        values_int_return[name] = returnValue;
+    }
+
```

Comment:
The idea is to call the method always `setReturn` and the method signature will be unique because of the different param types .